### PR TITLE
feat: Add booking info to departures

### PR DESCRIPTION
**Pull request by gorandalum**
Show booking info on:
- As info-box on The departure details screen, with booking action
buttons if currently bookable.
- As icon in the upper left corner (same as situations) in the
departures list v2.
- As right icon on the departure chip in departures list v1.

### CODE REVIEW

File: src/travel-details-screens/DepartureDetailsScreenComponent.tsx [hunk 1]
```diff
@@ -433,12 +440,24 @@ function EstimatedCallRow({
 
   const {group, isStartOfGroup, isEndOfGroup, isStartOfServiceJourney} =
     call.metadata;
+  const isFirstOfTrip = group === 'trip' && isStartOfGroup;
```
Comment (marius-at-atb) [inline src/travel-details-screens/DepartureDetailsScreenComponent.tsx:443]: Maybe this variable name could clarify what it is first of?

**Review by marius-at-atb — APPROVED** [review]
Lgtm! Really nice that you added tests for `statusComparator`, as it is not that easy to parse its internal logic when reading through 😄

File: src/travel-details-screens/DepartureDetailsScreenComponent.tsx (hunk 1 above)
Comment (gorandalum) [reply src/travel-details-screens/DepartureDetailsScreenComponent.tsx:443]: Do you have a suggestion here? Would it be better with `isFirstOfTripGroup`? Kinda hard to find better naming without diverging from the group terms ('passed', 'trip', 'after') we have in the code.

**Review by hanne-at-atb — APPROVED** [review]
LGTM ⭐

**Review by jorelosorio — APPROVED** [review]
Nice work, a see tests for it.

File: src/travel-details-screens/DepartureDetailsScreenComponent.tsx (hunk 1 above)
Comment (marius-at-atb) [reply src/travel-details-screens/DepartureDetailsScreenComponent.tsx:443]: Yea I think `isFirstOfTripGroup` would be better :+1:
Maybe also `isStartOfTripGroup` to keep the naming in line with `isStartOfGroup`?

**Pull request merged**